src/main/java/Duke.java

```diff
@@ -1,10 +1,255 @@
+import java.util.Scanner;
+
 public class Duke {
+
+    static final String HORIZONTAL_LINE = "____________________________________________________________";
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
-    }
-}
+
+        // Variables needed
+        Scanner scanner = new Scanner(System.in);
+        String userInput;
+        String command;
+
+        Task[] listOfInputs = new Task[100];
```

Comment:
Renaming the array to _tasks_ instead of _listOfInputs_ may give readers better understanding of its array type (array of class task)